importer/SeBbrSv.py

```diff
@@ -141,9 +139,11 @@ def set_no_of_buildings(self):
         extracted_no = utils.get_number_from_string(
             utils.get_text_inside_brackets(self.funktion))
         if extracted_no is not None:
+            ref = self.wlm_source
             self.add_statement(
                 "has_parts_of_class", "Q41176",
-                {"quantity": {"quantity_value": extracted_no}})
+                {"quantity": {"quantity_value": extracted_no}},
+                refs=[ref])
```

Comment:
Re: 1 -- I have only used single references so far, but I erred on the side of caution when making this... It does make sense to optimize for single references though.